oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/common/components/EventSubmitForm.jsx

```diff
@@ -9,7 +9,7 @@ import {
   RequestCommentInput,
 } from "@js/oarepo_requests_common";
 import { useMutation, useQueryClient } from "@tanstack/react-query";
-import { http } from "@js/oarepo_ui";
+import { http } from "react-invenio-forms";
 
 export const EventSubmitForm = ({
```

Comment:
This cmp should be renamed to CommentSubmitForm, as it deals just with comments, props api should be also reduced (why it needs to be tied to timeline or full request or page - form submission link should be enough necessary info for this form)

Reply:
Everything unrelated to the mission of this form (to submit a comment) must be handled by parent component where those concepts like timeline, pagination etc. live :)